packages/elements/src/components/ino-img/ino-img.scss

```diff
@@ -30,4 +29,34 @@ ino-img {
   &.mdc-image-list__item {
     display: list-item;
   }
+
+  img:before {
+    content: ' ';
+    display: block;
+    position: absolute;
+    width: inherit;
+    height: inherit;
+    background-color: #fff;
+  }
+
+  .ino-img__fallback-icon {
+    display: none;
+    position: absolute;
+    background-color: theme.color(light, light);
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+
+    ino-icon {
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      width: max(20px, 10%);
+      height: max(20px, 10%);
+      --icon-width: 100%;
+      --icon-height: 100%;
```

Comment:
The fallback icon is not fully centered because of those two lines. I think you can safely delete them.